blueprint/api.apib

```diff
@@ -169,6 +169,14 @@ You can list all comments or retrieve individual comments, as well as create and
 
     + Attributes (Comment Response)
 
++ Response 401 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (JSON Web Token Invalid Response)
+
++ Response 403 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Forbidden Response)
```

Comment:
If it's in the API Blueprint it should probably be tested.

Both tests and documentations are your contract that verify you're doing what you said you were going to do.